Sources/Apollo/FetchOptions.swift

```diff
@@ -0,0 +1,6 @@
+import Foundation
+
+public enum FetchOptions: String {
```

Comment:
I think it might be a little easier to understand this if it were something like `FetchHTTPMethod` - you're not offering any other options here, just letting the user set the HTTP method to use for a fetch.

Reply:
Totally agree. I'm changing it.